qa/tasks/vstart_runner.py

```diff
@@ -773,6 +773,28 @@ def list_connections():
 
         self.gather_mount_info()
 
+    def _set_fuse_daemon_pid(self):
```

Comment:
If we don't use `sudo`, is this necessary?

Reply:
Locally I have a CentOS 7 node, and the `sudo cmd` will fork 2 processes, such as:

```
root     13483  0.0  0.0 239152  7820 pts/0    S    10:29   0:00 /usr/bin/sudo -E LD_LIBRARY_PATH=/opt/rh/devtoolset-8/root/usr/lib64:/opt/rh/devtoolset-8/root/usr/lib:/opt/rh/devtoolset-8/root/usr/lib64/dyninst:/opt/rh/devtoolset-8/root/usr/lib/dyninst:/opt/rh/devtoolset-8/root/usr/lib64:/opt/rh/devtoolset-8/root/usr/lib:/data/test/ceph/build/lib/ PATH=/opt/rh/devtoolset-8/root/usr/bin:/usr/lib64/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin scl enable devtoolset-8  'nsenter' '--net=/var/run/netns/ceph-ns--tmp-tmpe_63x2wp-mnt.0' './bin/ceph-fuse' '-f' '--name' 'client.0' '/tmp/tmpe_63x2wp/mnt.0'
root     13503  0.0  0.0   4372  1484 pts/0    S    10:29   0:00 scl enable devtoolset-8  'nsenter' '--net=/var/run/netns/ceph-ns--tmp-tmpe_63x2wp-mnt.0' './bin/ceph-fuse' '-f' '--name' 'client.0' '/tmp/tmpe_63x2wp/mnt.0'
root     13520  0.2  0.2 1886960 38884 pts/0   Sl   10:29   0:00 ./bin/ceph-fuse -f --name client.0 /tmp/tmpe_63x2wp/mnt.0
```

Without this the vstart_runner.py will only kill the Process#13483, and the detached child Process#13503 and grandchild Process#13520 will keep alive, like:

```
root     13483  0.0  0.0 239152  7820 pts/0    S    10:29   0:00 [sudo] <defunct>
root     13503  0.0  0.0   4372  1484 pts/0    S    10:29   0:00 scl enable devtoolset-8  'nsenter' '--net=/var/run/netns/ceph-ns--tmp-tmpe_63x2wp-mnt.0' './bin/ceph-fuse' '-f' '--name' 'client.0' '/tmp/tmpe_63x2wp/mnt.0'
root     13520  0.2  0.2 1886960 38884 pts/0   Sl   10:29   0:00 ./bin/ceph-fuse -f --name client.0 /tmp/tmpe_63x2wp/mnt.0
```